### ISSUE TITLE
Tile: Moving tiles cannot be done using a keyboard (2.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#951]
### Expected behavior
A user can reorder/move tiles with their keyboard only.

### Actual behavior
There is no mechanism to support this. The move/drag and drop control in the tile header isn't an element type that can receive focus.
